content/intro-to-storybook/react/en/creating-addons.md

```diff
@@ -376,37 +395,46 @@ export const Content = () => {
     return null;
   }
 
+  const url = getUrl(results[selected]).replace("{id}", storyId);
+
   return (
     <Fragment>
       <Asset url={url} />
       {results.length > 1 ? (
         <ActionBar
           actionItems={results.map((i, index) => ({
-            title: typeof i === 'string' ? `asset #${index + 1}` : i.name,
-            onClick: () => setSelected(index),
+            title: typeof i === "string" ? `asset #${index + 1}` : i.name,
+            onClick: () => setSelected(index)
           }))}
         />
       ) : null}
     </Fragment>
   );
 };
-```
-
-## Extracting your addon into it's own package
 
-After your addon is working as intended, you will likely want to extract it into it's own package.
+addons.register("my/design-assets", () => {
+  addons.add("design-assets/panel", {
+    title: "assets",
+    type: types.PANEL,
+    render: ({ active, key }) => (
+      <AddonPanel active={active} key={key}>
+        <Content />
+      </AddonPanel>
+    )
+  });
+});
+```
 
-If it's useful to you, it's likely useful to someone else as well.
+</details>
 
-You can open a PR on the storybook documentation to have your addon featured there.
+## Next steps
 
-## What's next?
+The next logical step for our addon, would be to make it it's own package and allow it to be distributed with your team and possibly with the rest of the community.
 
-This is an example of an addon that uses parameters and displays it in a panel, but that's just one of the many options you have.
+But that's beyond the scope of this tutorial. This example demonstrates how you can use the Storybook API to create your own custom addon to further enchance your development workflow.
 
-With these principles you're able to display you custom UI in a variety of places in the storybook UI.
+Not only what we've accomplished here, but additionally you can also:
```

Comment:
"Learn how to further customize your addon:"

Reply:
I meant replace "Not only what we've accomplished here, but additionally you can also:" with "Learn how to further customize your addon:"

Reply:
i read it wrong, Sorry about that. I'll fix it and commit the change shortly.